stylesheets/_mixin.clear-fix.scss

```diff
@@ -1,13 +1,24 @@
-@mixin clear-fix() {
-  zoom: 1;
+@mixin clear-fix($simple: false) {
```

Comment:
Yeah, it was done this way on purpose so that all existing clear fixes continue to work as they always have.